roles/test_operator/tasks/stages.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: "Run stage {{ _stage_vars.name }}"
+  ansible.builtin.debug:
+    msg: "Run stage {{ _stage_vars.name }}"
+
+- name: "Call pre stage hooks for test operator stage {{ _stage_vars.name }}"
+  when:
+    - not cifmw_test_operator_dry_run | bool
+    - _stage_vars.pre_stage_hooks is defined
+  vars:
+    step: pre_test_hooks
+    hooks: "{{ _stage_vars.pre_stage_hooks }}"
+  ansible.builtin.import_role:
+    name: run_hook
+
+- name: Include stage var file
+  ansible.builtin.include_vars:
+    file: "{{ _stage_vars.test_file | default(ansible_user_dir ~ '/ci-framework-data/parameters/reproducer-variables.yml') }}"
```

Comment:
`reproducer-variables.yml` won't be available in non-reproducer environments like zuul. Anyway, even in a reproducer environment those variable would be already loaded. I suggest to run the include_vars if and only if `_stage_vars.test_file` is not empty.

Reply:
ack